plugins/missions.js

```diff
@@ -211,6 +335,7 @@ window.plugin.missions = {
 
 	zoomToMission: function(mission) {
 		map.fitBounds(this.getMissionBounds(mission), {maxZoom: DEFAULT_ZOOM});
+		// NOTE: Why maxZoom and not map.fitBounds(this.getMissionBounds(mission)); ???
```

Comment:
In order to prevent too close zoom

Reply:
The zoom level is too far out here in my opinion. I have removed the comment and will create an issue for this later on